apps/docs/content/guides/auth/auth-mfa.mdx

```diff
@@ -67,7 +58,7 @@ Adding MFA to your app involves these four steps:
    You need to provide a UI within your app that your users will be able to set-up
    MFA in. You can add this right after sign-up, or as part of a separate flow in
    the settings portion of your app.
-2. **Add unenrollment flow.**
+2. **Add unenroll flow.**
```

Comment:
Vale was complaining about `unenrollment` but it looks like `unenroll` doesn't pass either. Open to switching back to `unenrollment`